server.py

```diff
@@ -366,6 +367,7 @@ def node_info(node_class):
             obj_class = nodes.NODE_CLASS_MAPPINGS[node_class]
             info = {}
             info['input'] = obj_class.INPUT_TYPES()
+            info['input_order'] = {key: list(value.keys()) for (key, value) in obj_class.INPUT_TYPES().items()}
```

Comment:
Right now, the default front-end relies on the ordering of keys in a JSON object. While this is technically guaranteed in both Python and Javascript, the ordering is explicitly *not* guaranteed in the JSON standard itself. Technically, browsers and proxies are free to shuffle around the entries in a JSON object. More practically, languages that don't guarantee this ordering (like C++ or Go) would need to manually parse the results of this endpoint as a string rather than being able to use a standard JSON library.